proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -100,6 +100,10 @@ message DataCapture {
     // For compressed and GRPC bodies, if the size of the body is larger than this, we ignore
     // it entirely, otherwise we decompress/decode the body and then pass it to the filter.
     google.protobuf.Int32Value body_max_processing_size_bytes = 6;
+
+    // Array of allowed content type substrings to record
+    // ex: ["json"] will record any request bodies that have a content-type header that includes "json"
+    google.protobuf.StringValue allowed_content_types = 7;
```

Comment:
Should be `repeated`?

Reply:
Oh, right.  

Trying to update, for some reason setting this to repeated breaks our generated pbloader code.

```
go: found gopkg.in/yaml.v2 in gopkg.in/yaml.v2 v2.4.0
cd /Users/donniepropst/Projects/agent-config/gen/go && go test ./...
# github.com/hypertrace/agent-config/gen/go/v1 [github.com/hypertrace/agent-config/gen/go/v1.test]
v1/config.pbloader.go:214:19: syntax error: unexpected x at end of statement
FAIL    github.com/hypertrace/agent-config/gen/go/v1 [build failed]
FAIL
```

so need to sort that out

Reply:
@prodion23 usually you modify the templates https://github.com/hypertrace/agent-config/tree/f8a7187ff6ccb79bc94aa5583c9edfda81ac6f30/tools/go-generator/cmd/generator/_templates